authentication/models.py

```diff
@@ -58,6 +61,21 @@ def gamma(self) -> int:
             return 0  # see https://github.com/GrafeasGroup/blossom/issues/15
         return Submission.objects.filter(completed_by=self).count()
 
+    @property
+    def first_active(self) -> Optional[datetime.datetime]:
+        """
+        Return the date when the user was first active.
+
+        Usually, this is the date of their first transcription.
+
+        :return: the date when the user was first active or None if they haven't
+        transcribed yet.
+        """
+        first_claim = (
```

Comment:
Do  we know anything about the performance of this query? It sounds like it could be quite detrimental to execution time if there's many submissions, especially since they have to be sorted. Wouldn't we be better off just making this an additional field of the user, since this date does not change after the initial claim?

Reply:
I think it would be quite fast, but it's probably better to do it as @itsthejoker suggested and to just fix the `date_joined` field to actually be correct